ChiamakaObitube/AutoMart
# #165872549 User can post a car sale ad

**What does this PR do?**
This PR creates a page where user(seller) can post a car advertisement

**Description of Task to be completed?**
Have the following working:
ensure that when a user(seller) fills the form, an ad is posted for him and viewable on unsold cars page.

**How should this be manually tested?**
The following steps can be used:
Clone the repository or navigate to the App URL on github pages.
Signup or Signin with your user credentials.
The application will take you to the seller post ad page
Any background context you want to provide?
User should ensure there is internet on their machine to access the pages.

**What are the relevant pivotal tracker stories?**
#165872549
